sentry-android-ndk/src/main/jni/sentry.c

```diff
@@ -11,6 +11,90 @@ struct transport_options {
 
 struct transport_options g_transport_options;
 
+JNIEXPORT void JNICALL Java_io_sentry_android_ndk_NdkScopeObserver_nativeSetTag(JNIEnv *env, jclass cls, jstring key, jstring value) {
+    if (key == NULL) {
+        return;
+    }
+    const char *charKey = (*env)->GetStringUTFChars(env, key, 0);
+    const char *charValue = NULL;
+    if (value == NULL) {
+        charValue = (*env)->GetStringUTFChars(env, value, 0);
+    }
+    sentry_set_tag(charKey, charValue);
+}
+
+JNIEXPORT void JNICALL Java_io_sentry_android_ndk_NdkScopeObserver_nativeSetUser(
```

Comment:
L: missing the `other` field.

Reply:
Yes, I wanted to simplify this for now to unblock folks